source/style-guide/writing/write-clear-concise-sentences-paragraphs.txt

```diff
@@ -54,56 +54,127 @@ Use a Consistent Sentence Structure
+       restore files to their original folders. Click :guilabel:`Next`.
      - Click the check boxes to confirm whether you would like to
        Overwrite files with the same name or restore the files to their
-       original folders and then click the **Next** button.
+       original folders and then click the :guilabel:`Next` button.
+
+.. _sg-limit-verb-be:
+
+Limit Using *Be* as the Main Verb
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pay special attention if a form of be is the main verb in the sentence.
+Forms of *be* include:
+
+- *be*
+- *is*
+- *am*
+- *are*
+- *was*
+- *were*
+- *being*
+- *been*
+
+Look for a "hidden" verb in the element following the *be*‐verb.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - This report analyzes the problem.
+     - This report is an analysis of the problem.
+
+.. _sg-avoid-expletive-structures:
+
+Avoid Expletive Structures
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An expletive sentence begins with *It* or *There* and followed by a form of *be*. Consider another noun and verb following the expletive.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - We need to solve this problem.
+     - It is necessary for us to find a solution to this problem.
+
+   * - There are three benefits to the cloud backup compared to the
+       legacy backup.
+     - Cloud backup has three benefits over legacy backup.
+
+.. _sg-avoid-nominalizations:
+
+Avoid Nominalizations
+~~~~~~~~~~~~~~~~~~~~~
+
+A *nominalization* is a noun derived from a verb. These words often end
+in -ance, -ence, -ion, or -ment. Identify the verb at the root of the
+nominalization and use it instead.
+
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Use
+     - Avoid
+
+   * - The modal recommends upgrading to new software.
+     - The modal contains a recommendation that the user upgrade to new
+       software.
+
+   * - It would benefit the users if they had this information.
+     - It would be beneficial to the users if they had this
+       information.
 
 .. _use-necessary-words:
 
 Use the Necessary Words
-~~~~~~~~~~~~~~~~~~~~~~~
+-----------------------
 
 Business and technical writing isn't creative writing.
 
-- Use only the words necessary to convey the meaning.
-- Strip out anything extraneous.
+- Limit yourself to the words necessary to convey the meaning.
+- Remove anything extraneous.
 
-  - Are you using adverbs (modifiers ending in *-ly*)? If so, you can
-    remove most of them without changing the meaning.
-  - What about adjectives? If they aren't necessary to the meaning,
-    remove them.
-  - Can prepositional phrases be shortened?
-  - Are you using empty phrases that don't clarify the content?
+  - Are you using adverbs (modifiers ending in *-ly*)? You can remove
+    most adverbs without changing the meaning.
+  - Are you using adjectives? Remove them if they aren't necessary to
+    the meaning.
```

Comment:
Some features include adjectives as part of their name. Be careful that you are not removing an adjective that functions as a vital identifier for a feature.  "Resumable Initial Sync" cannot be called anything else, for example, 

Generally, Jira tickets will be careful to call this out, either in the Scope doc, of even as a feature branch or Jira component thing.

Further, adjectives serve to differentiate cases where a statement applies ... and where it expressly doesn't. With this one:

Indexes Build Simultaneously on "Data-Bearing Replica Set Members"

Removing any of the adjs will render the statement misleading at best, or untrue at worst. (differing behavior for primary + secondaries vs arbiters vs non-data-bearing vs standalone vs sharded cluster members)

General gist of this is spot on: No one cares that the Resumable Initial Sync showed up to work in a blue suit on a cloudy day. But we can't call it, itself, anything different than what it is.